{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2019-19232",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "sudo",
            "product": {
              "product_data": [
                {
                  "product_name": "sudo",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "1.8.29",
                        "version_affected": "<="
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "NVD-CWE-noinfo"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://access.redhat.com/security/cve/cve-2019-19232",
          "name": "https://access.redhat.com/security/cve/cve-2019-19232",
          "refsource": "CONFIRM",
          "tags": [

          ]
        },
        {
          "url": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs58103",
          "name": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs58103",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs58812",
          "name": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs58812",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs58979",
          "name": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs58979",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs76870",
          "name": "https://quickview.cloudapps.cisco.com/quickview/bug/CSCvs76870",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://security.netapp.com/advisory/ntap-20200103-0004/",
          "name": "https://security.netapp.com/advisory/ntap-20200103-0004/",
          "refsource": "CONFIRM",
          "tags": [

          ]
        },
        {
          "url": "https://support2.windriver.com/index.php?page=cve&on=view&id=CVE-2019-19232",
          "name": "https://support2.windriver.com/index.php?page=cve&on=view&id=CVE-2019-19232",
          "refsource": "CONFIRM",
          "tags": [

          ]
        },
        {
          "url": "https://support2.windriver.com/index.php?page=defects&on=view&id=LIN1018-5506",
          "name": "https://support2.windriver.com/index.php?page=defects&on=view&id=LIN1018-5506",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://www.bsi.bund.de/SharedDocs/Warnmeldungen/DE/CB/2019/12/warnmeldung_cb-k20-0001.html",
          "name": "https://www.bsi.bund.de/SharedDocs/Warnmeldungen/DE/CB/2019/12/warnmeldung_cb-k20-0001.html",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://www.sudo.ws/devel.html#1.8.30b2",
          "name": "https://www.sudo.ws/devel.html#1.8.30b2",
          "refsource": "CONFIRM",
          "tags": [
            "Vendor Advisory"
          ]
        },
        {
          "url": "https://www.sudo.ws/stable.html",
          "name": "https://www.sudo.ws/stable.html",
          "refsource": "MISC",
          "tags": [
            "Vendor Advisory"
          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "** DISPUTED ** In Sudo through 1.8.29, an attacker with access to a Runas ALL sudoer account can impersonate a nonexistent user by invoking sudo with a numeric uid that is not associated with any user. NOTE: The software maintainer believes that this is not a vulnerability because running a command via sudo as a user not present in the local password database is an intentional feature. Because this behavior surprised some users, sudo 1.8.30 introduced an option to enable/disable this behavior with the default being disabled. However, this does not change the fact that sudo was behaving as intended, and as documented, in earlier versions."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:a:sudo:sudo:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "1.8.29"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "AV:N/AC:L/Au:N/C:N/I:P/A:N",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "PARTIAL",
        "availabilityImpact": "NONE",
        "baseScore": 5.0
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 10.0,
      "impactScore": 2.9,
      "acInsufInfo": false,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2019-12-19T21:15Z",
  "lastModifiedDate": "2020-01-30T16:15Z"
}